src/libpriv/rpmostree-origin.cxx

```diff
@@ -714,6 +722,7 @@ rpmostree_origin_remove_packages (RpmOstreeOrigin  *origin,
     update_keyfile_pkgs_from_cache (origin, "packages", "requested-local",
                                     origin->cached_local_packages, TRUE);
 
+  sync_baserefspec (origin);
```

Comment:
Isn't this redundant with the calls we do in `update_keyfile_pkgs_from_cache`?

Reply:
Yeah it is.  Doesn't seem worth rerunning all the CI just for that, I'll fix it in https://github.com/coreos/rpm-ostree/pull/2743 ?